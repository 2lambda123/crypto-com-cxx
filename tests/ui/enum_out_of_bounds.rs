#[cxx::bridge]
mod ffi {
    #[repr(u32)]
    enum Bad {
        A = 0xFFFF_FFFF_FFFF_FFFF,
    }
}

fn main() {}
