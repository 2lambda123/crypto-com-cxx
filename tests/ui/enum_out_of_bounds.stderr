error: discriminant value `18446744073709551615` is outside the limits of u32
 --> $DIR/enum_out_of_bounds.rs:5:9
  |
5 |         A = 0xFFFF_FFFF_FFFF_FFFF,
  |         ^^^^^^^^^^^^^^^^^^^^^^^^^
